pvlib/pvsystem.py

```diff
@@ -1372,6 +1398,67 @@ def get_iam(self, aoi, iam_model='physical'):
         else:
             raise ValueError(model + ' is not a valid IAM model')
 
+    def dc_ohms_from_percent(self):
+        """
+        Calculates the equivalent resistance of the wires using
+        :py:func:`pvlib.pvsystem.dc_ohms_from_percent`
+
+        Makes use of array module parameters according to the
+        following DC models:
+        CEC:
+            `self.module_parameters["V_mp_ref"]`,
+            `self.module_parameters["I_mp_ref"]`,
+        SAPM:
+            `self.module_parameters["Vmpo"]`,
+            `self.module_parameters["Impo"]`,
+        PVsyst-like or other:
+            `self.module_parameters["Vmpp"]`,
+            `self.module_parameters["Impp"]`,
+
+        Other array parameters that are used are:
+        `self.losses_parameters["dc_ohmic_percent"]`,
+        `self.modules_per_string`, and
+        `self.strings`.
```

Comment:
Can you add some blank lines to improve how this is rendered? rst is finicky about this kind of thing. I think you'll want a blank line after every `:`. You'll also need `*` if you want the parameters to appear in a list. For example:

```
        Makes use of array module parameters according to the
        following DC models:

        CEC:

            * `self.module_parameters["V_mp_ref"]`,
            * `self.module_parameters["I_mp_ref"]`,
```